cmd/kpod/save.go

```diff
@@ -81,9 +85,19 @@ func saveCmd(c *cli.Context) error {
 // saveImage pushes the image to docker-archive or oci by
 // calling pushImage
 func saveImage(store storage.Store, opts saveOptions) error {
-	dst := dockerArchive + opts.output
+	var dst string
+	switch opts.format {
+	case libkpodimage.OCIArchive:
+		dst = libkpodimage.OCIArchive + ":" + opts.output
+	case libkpodimage.DockerArchive:
```

Comment:
```
case  libkpodimage.DockerArchive:
       fallthrough
case "":
      	dst = libkpodimage.DockerArchive + ":" + opts.output 
```

Reply:
Didn't know this was a thing, learnt something today :) Fixed